pyaedt/modeler/Primitives.py

```diff
@@ -107,14 +107,12 @@ class Polyline(Object3d):
 
     Parameters
     ----------
-    parent : :class:`pyaedt.modeler.Primitives.Primitives`
+    primitives : :class:`pyaedt.modeler.Primitives.Primitives`
         Pointer to the parent Primitives object.
```

Comment:
Is "Primitives" necessary in the description? It is not included in line 812 below.

Reply:
yes it is. Thanks